### DISCUSSION TITLE
Terminal Error during test

### DISCUSSION THREAD

**Discussion by MaCJaX** [thehighnibble/thehighnibble.github.io#134] [IMSAI8080esp replica]
Hello, I assembled the IMSAI 8080 replica kit. I have all the IC's installed in their correct locations and orientations however, I connect to the ESP32 via Putty and I am getting a constant reboot along with the attached error in the screen shot. Any ideas how to fix this error? Thanks in advance!
![ESP32 err](https://github.com/thehighnibble/thehighnibble.github.io/assets/35588650/f44036f6-9d79-4ac7-a205-ac504fa5337d)

**Comment by nbreeden2** [comment]
This looks like there is an issue reading the PSRAM. This is the little 8-pin surface mount IC. Check all the pins for good solder joints and also look for solder bridges between the pins. I need to use magnification to be really sure all 8 pins are soldered well and there are no bridges.

**Reply by MaCJaX** [reply]
Thanks! Ill check it out.

**Reply by MaCJaX** [reply]
Ok so I checked out that IC and it looks good. I attached a pic. I am thinking there is a problem with the SD card reader? I have some new ones on order but it seems like pin 3 is connected to ground on it. Not sure if it supposed to be like that but judging by some drawings I see online, it doesn't seem like it should be. Soldering on it looked good so I just removed it and will wait for the replacements.
![ram](https://github.com/thehighnibble/thehighnibble.github.io/assets/35588650/cfff6815-0f5d-4499-a260-84214a0f3892)

**Comment by nbreeden2** [comment]
You commented the SD card reader has pin 3 connected to VSS/Ground. Are you sure you have the pinout correct. I believe this is the correct pinout. The two traces that are wider should be the power traces.
![bridge](https://github.com/thehighnibble/thehighnibble.github.io/assets/37715892/2e4ec8e9-1ef1-4d9e-aed3-8bc0f1ccffdc)

**Reply by MaCJaX** [reply]
When testing for shorts, I had continuity between VSS/Ground to the metal case so I thought maybe that was an issue but it looks like that is correct. I said pin 3 but it's actually the pin 6.

**Comment by MaCJaX** [comment]
Ok, so I installed new SD card socket and removed and resolder the PSRAM but still same error and ESP32 constantly reboots. Not sure what else to check? SD card contents look good. It's formatted FAST32 and I see the proper folder structure.

**Comment by nbreeden2** [comment]
Did you solder the 6 extra pins onto the ESP32 (3 pins per side). I know missing these causes issues though I don't recall the specifics.

**Reply by MaCJaX** [reply]
oh, no. Ha, I missed those. I was wondering why there was only some soldered so I figured that's how it's should be. Just soldered them on and it works great now! Thanks so much !!!

**Comment by nbreeden2** [comment]
That's great news!
